src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import StreamGroupSerializer
+from sentry.models import (
+    Group, GroupCommitResolution, Project,
+    Release, ReleaseCommit,
+)
+
+
+class IssuesResolvedInReleaseEndpoint(OrganizationEndpoint):
```

Comment:
for this endpoint, we should probably either inherit from the `ProjectEndpoint` so that we're actually checking project permissions (since we're returning group data) (or maybe even the `GroupEndpoint`?)